nbconvert/exporters/pdf.py

```diff
@@ -163,9 +163,9 @@ def from_notebook_node(self, nb, resources=None, **kw):
         )
         # set texinputs directory, so that local files will be found
         if resources and resources.get('metadata', {}).get('path'):
-            self.texinputs = resources['metadata']['path']
+            self.texinputs = str_to_unicode(resources['metadata']['path'])
         else:
-            self.texinputs = os.getcwd()
+            self.texinputs = str_to_unicode(os.getcwd())
```

Comment:
The py3compat module should have a function `getcwd` which points to `os.getcwdu()` on Python 2, which is neater than converting.